include/osquery/dispatcher.h

```diff
@@ -99,20 +60,13 @@ class InterruptableRunnable {
    * Interruption means resources have been stopped.
    * Non-interruption means no attempt to affect resources has been started.
    */
-  Mutex stopping_;
+  std::mutex stopping_;
```

Comment:
This is a nit, but I was recommending people to use the `Mutex` alias for consistency.

Reply:
Mutex is read-write lock, where both read and write locking mechanisms are a lot more complicated. I don't see any reason to use it if we get advantage of the both read and write lock.
What do you think about that?